e2e/config/create-user.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+#
+# Copyright 2023 Gravitational, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+# Check if the user is already created
+if [ -f /var/lib/teleport/user-create ]; then
+    echo "User already created"
+    exit 0
+fi
+
+echo "Creating user"
+
+# Give the auth server some time to start
+sleep 5
+
+# Create the user
+tctl -d users add bob --roles=access,editor --logins=root
+# Set the password - 'secret'.
+# TODO(jakule): I was not able to figure out how to set the password using WebUI in automated way, so I'm doing it directly in the database.
+sqlite3 /var/lib/teleport/backend/sqlite.db "UPDATE kv SET value='\$2a\$10\$w0K2pwK/cF8BG0kKZ7X1qe1uU7w3mwZ2S46PO6SlaiVjiTkbNGQp6' where key = '/web/users/bob/pwd';"
```

Comment:
I'll change that to use `tctl create` to create users.

Reply:
Not needed after applying this suggestion https://github.com/gravitational/teleport/pull/29286#discussion_r1273864120